api/src/gameState/gameState.controller.ts

```diff
@@ -162,4 +169,18 @@ export class GameStateController {
     this.gatewayService.emitGameState(newGameState);
     return newGameState;
   }
+
+  @Delete(':id')
+  async deleteGame(@Param('id', ParseIntPipe) id: number) {
+    return this.gameStateService.deleteGame({ id });
+  }
+
+  @Delete()
+  async deleteManyGame(@Query('filter') filter: any) {
+    if (!filter) {
+      throw new HttpException('Missing parameter', 400);
```

Comment:
Just for you to know, you could use the Validation mecanism of NestJs, I haven't really tried it, but it seems to be done for that. (https://docs.nestjs.com/techniques/validation#using-the-built-in-validationpipe)